### ISSUE TITLE
a well-deserved public apology to felix fietkau

### ISSUE THREAD

**Issue by i3roly** [openwrt/mt76#536]
hello everyone,

in case you missed it, i made a rather large scene about my displeasure with the mt76 driver some weeks ago (see https://www.github.com/openwrt/mt76/issues/521).

i went about my unhappiness with the mt76 driver the wrong way, no matter how passionate it may have been.

in that time, i have spoken to felix and i retract my original assertions that "fefelix" of #openwrt on freenode was the same person.

originally, i had spoken to felix shortly after being banned from participating on openwrt-related github activities. this did not bother me much. 

BUT in this same time period, i have benefited by soliciting and leveraging the knowledge of the openwrt userbase on the forums, even though it was pretty obvious i was doing so for "extracurricular activities". i was never banned--in fact, one of the moderators had actually discussed my 'flagged' posts with the others and came to the (accurate) conclusion that i found these posts humorous.

i must say i feel more at home on the openwrt forums than i do the dd-wrt forums, and i don't really miss the latter.

even in the past couple of days i bothered felix about the horrific (it really is BAD!) LANTIQ U-boot and he tried to help me the best he could, but shared the insight that TP-LINK bootloaders are also different (on top of the horawful LANTIQ variant).

in short, i am issuing an apology and i retract any uncertainty i had (i did have some in the following week or so, then i didn't think about it) regarding fefelix and "nbd" (or felix fietkau) being distinct entities.

i want to thank the openwrt community for accommodating my unusual conduct and i hope to continue to be a part of the scene.

**Comment by dangowrt** [comment]
a well-deserved public apology to felix fietkau
